3.interpret_models/linear_reg_plate_1_2_norm_data/feature_importances/nbconverted/seperate_compartments.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Here the features are seperated according to compartments for later analysis
+
+# In[1]:
+
+
+# %%--%%| <qGnYViiwRD|SyZ3qa8iz3>
+r"""°°°
+## Imports
+°°°"""
+
+
+# In[ ]:
+
+
+import pandas as pd
+import numpy as np
+from pathlib import Path
+from sklearn.preprocessing import LabelEncoder
+from sklearn.linear_model import LogisticRegression
+
+from sklearn.metrics import (
+    confusion_matrix,
+    ConfusionMatrixDisplay,
+    precision_score,
+    accuracy_score,
+)
+from sklearn.model_selection import train_test_split
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+import itertools
+
+from joblib import dump, load
+
+
+# ## Find the git root Directory
+
+# In[ ]:
+
+
+# Get the current working directory
+cwd = Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
+
+# Check if a Git root directory was found
+if root_dir is None:
+    raise FileNotFoundError("No Git root directory found.")
+
+
+# ## Create the output path if it doesn't exist
+
+# In[ ]:
+
+
+output_path = Path("data")
+
+output_path.mkdir(
+    parents=True, exist_ok=True
+)  # Create the parent directories if they don't exist
+
+
+# ## Import the model data as a dataframe
+
+# In[ ]:
+
+
+feature_properties = pd.read_csv(
+    root_dir
+    / "1.train_models/linear_reg_plate_1_2_norm_data/data"
+    / "model_properties.tsv",
+    sep="\t",
+)
+
+
+# ## Seperate cell data by channel
```

Comment:
It might be useful here (or maybe even in the first cell of this notebook) to describe your procedure for splitting compartments in full summary detail. It will also help to provide rationale in this initial statement for why splitting is necessary.

Reply:
Good idea, I just added a statement.